packages/hint-no-vulnerable-javascript-libraries/src/hint.ts

```diff
@@ -76,9 +81,19 @@ export default class NoVulnerableJavascriptLibrariesHint implements IHint {
         const loadSnyk = async () => {
             const oneDay = 3600000 * 24;
             const now = Date.now();
-            const snykDBPath = path.join(__dirname, 'snyk-snapshot.json');
+            const snykDBPath = require.resolve('./snyk-snapshot.json');
             let snykStat: fs.Stats;
 
+            /*
+             * We check if requestAsync exists, because if
+             * the browser extension is running this hint, then
+             * `requestAsync` will be null and we can't download
+             * the file.
+             */
+            if (!requestAsync) {
+                return loadJSONFileAsync(snykDBPath);
```

Comment:
For this case, can we just `require('./snyk-snapshot.json')`? That's how I handled this for `parser-manifest` to simplify bundling because `webpack` picks this up and includes it automatically (so you won't need the new `loadJSONFileAsync` abstraction).